submissions/Shrai-dev/html-movie-seat-booking/style.css

```diff
@@ -0,0 +1,442 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  height: 100%;
+  width: 100%;
+  background-image: url('./assets/background-image.jpg');
+  background-size: cover;
+  background-repeat: no-repeat;
+  font-family: 'Balsamiq Sans', cursive;
+  font-family: 'Montserrat', sans-serif;
+}
+
+.main {
+  margin: 30px 50px;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  background-color: #f1f1f1;
+  opacity: 0.8;
+  border-radius: 30px;
+}
+
+.movie__info {
+  margin-top: 20px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.movie__image {
+  height: 450px;
+  width: 300px;
+  margin: 10px 20px;
+}
+
+.movie__description {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  margin: 20px;
+}
+
+.movie__title {
+  margin: 5px;
+  font-size: 22px;
+}
+
+.movie__infolist {
+  list-style-type: none;
+  margin: 5px;
+  padding: 5px;
+}
+
+.movie__infolist-item {
+  padding: 5px;
+  font-size: 16px;
+}
+
+.movie-hall {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  margin-top: -150px;
+}
+
+.seats__info {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+}
+
+.seats__info-text {
+  padding: 10px;
+  font-size: 16px;
+  text-align: center;
+}
+
+.seats-taken {
+  position: relative;
+}
+
+.seats-taken::before {
+  content: '';
+  position: absolute;
+  margin-left: -30px;
+  width: 20px;
+  height: 20px;
+  border-radius: 4px;
+  box-shadow: 0 0 0 0.1em #6f6f70;
+  background-color: #c4c1c1;
+}
+
+.seats-free {
+  position: relative;
+  margin-left: 30px;
+}
+
+.seats-free::before {
+  content: '';
+  position: absolute;
+  margin-left: -30px;
+  width: 20px;
+  height: 20px;
+  border-radius: 4px;
+  background-color: #ffffff;
+  box-shadow: 0 0 0 0.1em #6f6f70;
+}
+
+.seats-chosen {
+  position: relative;
+  margin-left: 30px;
+}
+
+.seats-chosen::before {
+  content: '';
+  position: absolute;
+  margin-left: -30px;
+  width: 20px;
+  height: 20px;
+  border-radius: 4px;
+  background-color: #3ed145;
+  box-shadow: 0 0 0 0.1em #6f6f70;
+}
+
+.screen__title {
+  margin-top: 20px;
+  text-align: center;
+  font-size: 20px;
+}
+
+.screen__title::after {
+  content: '';
+  display: flex;
+  justify-content: center;
+  margin-left: 45px;
+  -moz-margin-start: 25px;
+  margin-bottom: 30px;
+  width: 420px;
+  border-bottom: 3px solid #000000;
+}
+
+.seats-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  margin: 10px 0;
+}
+
+.row {
+  flex-direction: row;
+  padding: 3px;
+  margin-right: 82px;
+}
+
+.seat__option {
+  margin-bottom: 0.5em;
+}
+
+.last {
```

Comment:
Not necessary to use extra class for it, Read more about pseudoclass `[:last-child`](https://developer.mozilla.org/en-US/docs/Web/CSS/:last-child) and [`:last-of-type`](https://developer.mozilla.org/en-US/docs/Web/CSS/:last-of-type) and choose one of them.